suite-native/app/src/App.tsx

```diff
@@ -22,11 +22,13 @@ import { useFormattersConfig } from './hooks/useFormattersConfig';
 const AppComponent = () => {
     const dispatch = useDispatch();
     const formattersConfig = useFormattersConfig();
+    const [isConnectInitialized, setIsConnectInitialized] = useState(false);
 
     useEffect(() => {
         const initActions = async () => {
             await dispatch(connectInitThunk()).unwrap();
             await dispatch(initBlockchainThunk()).unwrap();
+            setIsConnectInitialized(true);
```

Comment:
Is really necessary to have this after `initBlockchainThunk`? What if you place it right after `connectInitThunk`? I can imagine that blockchain thunk can sometimes take quite a long to finish.

Also please add some fallback in case of error, because otherwise it will hang on splash screen indefinitely which will be very confusing and hard to debug for us.

Reply:
821166982cc90e0934ecc2a3a8f19d24b422345a